1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -33,7 +32,7 @@ A transpiler would analyze our code and rewrite `height ?? 100` into `(height !=
 height = height ?? 100;
 
 // after running the transpiler
-height = (height !== undefined && height !== null) ? height : 100;
```

Comment:
Sure, these two lines do the exact same thing, but since the old one is much easier to understand, it's better. If some parentheses make code easier to read, they should stay. I don't approve of this change.